collectors/CookiePopupCollector.js

```diff
@@ -58,14 +62,16 @@ class CookiePopupCollector extends BaseCollector {
         session.on('Runtime.executionContextCreated', async ({context}) => {
             // ignore context created by puppeteer / our crawler
             if (!context.origin || context.origin === '://' || context.auxData.type !== 'default') {
+                if (context.auxData.type === 'isolated' && context.name.startsWith(ISOLATED_WORLD_PREFIX)) {
```

Comment:
Can `auxData.type` be anything other than `default` or `isolated`? Why do these two `if`s differ to `CMPCollector`? Do we need two `if`s? It’s difficult to understand the logic here.

Reply:
it can also be `worker`. I cleaned it up in the ContentScriptCollector